conda_pack/core.py

```diff
@@ -660,9 +660,32 @@ def find_site_packages(prefix):
 
 
 def check_no_editable_packages(prefix, site_packages):
-    pth_files = glob.glob(os.path.join(prefix, site_packages, '*.pth'))
     editable_packages = set()
+
+    # Check for modern editable packages (pip >= 20.0)
+    # Look for __editable__*.pth files
+    modern_pth_files = glob.glob(os.path.join(prefix, site_packages, '__editable__*.pth'))
```

Comment:
Instead, look inside `site-packages/*.dist-info/direct_url.json` for `{"dir_info": {"editable": true}, "url": "file:///Users/dholth/prog/conda-index"}`. Not all packaging backends will produce `__editable__*.pth`.
Of course, it's possible to have older "no direct_url.json" editable packages, or similar-looking `.pth` files that aren't following recent packaging conventions.
https://packaging.python.org/en/latest/specifications/direct-url/

Reply:
@dholth, would you say this is the best guarantee we have to see if something is an editable install?  I ask because I kept the old way and also add this new way but now, me thinks, it may not be worth it to look for those types of files at all.

Reply:
Cool, i will take it out then.